pkg/lib/k8s/k8s.go

```diff
@@ -46,16 +46,17 @@ var (
 )
 
 type Client struct {
-	RestConfig       *kclientrest.Config
-	clientset        *kclientset.Clientset
-	podClient        kclientcore.PodInterface
-	serviceClient    kclientcore.ServiceInterface
-	configMapClient  kclientcore.ConfigMapInterface
-	deploymentClient kclientapps.DeploymentInterface
-	jobClient        kclientbatch.JobInterface
-	ingressClient    kclientextensions.IngressInterface
-	hpaClient        kclientautoscaling.HorizontalPodAutoscalerInterface
-	Namespace        string
+	RestConfig         *kclientrest.Config
+	clientset          *kclientset.Clientset
+	podClient          kclientcore.PodInterface
+	serviceClient      kclientcore.ServiceInterface
+	istioServiceClient kclientcore.ServiceInterface
```

Comment:
I think it would be better to make a second `lib.k8s.Client` in the "istio-system" namespace in the operator package, and keep app-specific things out of `lib/k8s`, what do you think?

Reply:
agreed